backend/app/services/data_service.py

```
@@ -113,7 +113,7 @@ def load_data(self, symbol: str, source: str = None) -> pd.DataFrame:
                 # If no local data found, try to fetch it
                 logger.info(f"No local data found for {symbol}, fetching from API...")
                 # Fetch data from API
-                info = self.fetch_data(symbol=symbol, source=source or "coingecko", days=365)
+                info = self.fetch_data(symbol=symbol, source=source or "coinmarketcap", days=365)
```

Ruff (0.11.9)

Local variable `info` is assigned to but never used

Remove assignment to unused variable `info`

(F841)